<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comment_screen</class>
 <widget class="QWidget" name="comment_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>494</width>
        <height>359</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="write_comment_section">
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="write_comment_button">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Write a comment</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="comments_section">
           <item row="1" column="0">
            <widget class="QPushButton" name="comments_button">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Load Comments</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
